File: Iniciante/URI 1041.java
// https://www.beecrowd.com.br/judge/pt/problems/view/1041

import java.io.IOException;
import java.util.Scanner;
public class Main {
    public static void main(String[] args) throws IOException {
        Scanner scanner = new Scanner(System.in);
        double X = scanner.nextDouble();
        double Y = scanner.nextDouble();
        if (X == 0.0 && Y == 0.0)
        {
        System.out.print("Origem\n");
        }
        else if (X == 0.0 && Y != 0.0)
        {
        System.out.print("Eixo Y\n");
        }
        else if (Y == 0.0 && X != 0.0)
        {
        System.out.print("Eixo X\n");
        }
        else if (X > 0.0)
    {
        if (Y > 0.0)
        {
            System.out.print("Q1\n");
        }
        else System.out.print("Q4\n");
    }
        else if (Y > 0.0)
    {
        System.out.print("Q2\n");
    }
        else System.out.print("Q3\n");
    }
 
}

File: Iniciante/URI 1041.py
# https://www.beecrowd.com.br/judge/pt/problems/view/1041

X, Y = map(float, input().split())
if (X == 0.0 and Y == 0.0):
    print("Origem")
elif (X == 0.0 and Y != 0.0):
    print("Eixo Y")
elif (Y == 0.0 and X != 0.0):
    print("Eixo X")
elif (X > 0.0):
    if (Y > 0.0):
        print("Q1")
    else: print("Q4")
elif (Y > 0.0):
    print("Q2")
else: print("Q3")
